README.md

```diff
@@ -127,7 +127,34 @@ The following dependency is **only relevant if you intend to contribute** to the
 
 The following steps are **only relevant if you intend to contribute** to the repository. They are **not required** for agent development.
 
-- To install development dependencies (here optionally skipping `Pipfile.lock` creation):
+### Recommended
```

Comment:
what about building docs?

Reply:
also, it seems something like `docs_serve` is missing. Maybe worth explaining how to actually serve docs so to explore them?

Reply:
That's mentioned further down..